plugin/efficientNMSPlugin/README.md

```diff
@@ -98,6 +98,7 @@ The following four output tensors are generated:
 |`int`     |`max_output_boxes`        |The maximum number of detections to output per image.
 |`int`     |`background_class`        |The label ID for the background class. If there is no background class, set it to `-1`.
 |`bool`    |`score_activation` *      |Set to true to apply sigmoid activation to the confidence scores during NMS operation.
+|`bool`    |`class_agnostic`          |Set to true to do class-independent nms, otherwise, different class would do nms respectively.
```

Comment:
```suggestion
|`bool`    |`class_agnostic`          |Set to true to do class-independent NMS; otherwise, boxes of different classes would be considered separately during NMS.
```